From d420df0e69cd3414c58cf97ac2dcedd4e87836da Mon Sep 17 00:00:00 2001
From: Youssef DALIL <youssef.dalil@elsys-design.com>
Date: Mon, 8 Nov 2021 15:42:03 +0100
Subject: [PATCH] Commit to generate patch


diff --git a/hellopatch/files/hellopatch.c b/hellopatch/files/hellopatch.c
index fd15258..b3e14b7 100644
--- a/hellopatch/files/hellopatch.c
+++ b/hellopatch/files/hellopatch.c
@@ -3,6 +3,6 @@
 int main (void){
 
 	printf("Hello hellopatch \n");
-
+	printf("Generate patch...\n");
 	return 0;
 }
-- 
2.25.1

