From 4182a20a9cd87844b48c9267580e898be867321f Mon Sep 17 00:00:00 2001
From: Youssef DALIL <youssef.dalil@elsys-design.com>
Date: Mon, 22 Nov 2021 10:05:55 +0100
Subject: [PATCH] First patch

---
 hellopatch/files/test2.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hellopatch/files/test2.c b/hellopatch/files/test2.c
index 304b69f..2805c08 100644
--- a/hellopatch/files/test2.c
+++ b/hellopatch/files/test2.c
@@ -3,5 +3,6 @@
 int main (void) {
 
 	printf("This is a test \n");
+	printf("This is a first patch test \n");
 	return 0;
 }
-- 
2.25.1

